.scalafmt.conf

```
@@ -21,5 +22,8 @@ project.excludeFilters = [
 fileOverride {
   "glob:**/scala-3*/**" {
     runner.dialect = scala3
+    rewrite.scala3.convertToNewSyntax = yes
+    rewrite.scala3.removeOptionalBraces = yes
+    rewrite.scala3.insertEndMarkerMinLines = 30
```

```suggestion
    rewrite.scala3.insertEndMarkerMinLines = 10
```
maybe a smaller number?

---

I don't mind. Could you check locally if having it 10 it's ok for you?
Initially I set 20 but that there were much more markers than I would put manually.

---

From personal experience, I copypaste the same config to all of my Scala 3 projects:

```
fileOverride {
  "glob:**/*.scala" {
    align.preset = more
    runner.dialect = scala3
    rewrite.scala3.insertEndMarkerMinLines = 15
    rewrite.scala3.removeOptionalBraces = true
    rewrite.scala3.convertToNewSyntax = true
  }
}
```

And fairly often I desire to have 10 lines (for super nested functions).

---

@keynmol thx for sharing! Changed to 15